packages/@ourworldindata/grapher/src/mapCharts/MapTooltip.tsx

```diff
@@ -103,25 +104,22 @@ export class MapTooltip extends React.Component<MapTooltipProps> {
         return this.hasTimeSeriesData
     }
 
-    @computed private get tooltipTarget(): {
-        x: number
-        y: number
-        featureId: string
-    } {
-        return (
-            this.props.tooltipTarget ?? {
-                x: 0,
-                y: 0,
-                featureId: "Default Tooltip",
-            }
-        )
-    }
-
     // Line chart fields
     @computed private get sparklineTable(): OwidTable {
         return this.timeSeriesTable ?? new OwidTable()
     }
     @computed private get sparklineManager(): LineChartManager {
+        // use the whole time range for the sparkline, not just the range where this series has data
+        let { minTime, maxTime } = this.props.timeSeriesTable ?? {}
```

Comment:
`minTime`/`maxTime` can extend outside the range of times for which data exists. Is there an easier way than bracketing `getTimesUniqSortedAscForColumns` to get the same start & end times as the slider control uses?

Reply:
It needs to have rounded corners though or else it draws a square box that's not clipped by the container's rounded corners